frontend/lib/rh/rental-history.tsx

```diff
@@ -214,88 +226,94 @@ function RentalHistoryPreview(): JSX.Element {
         </p>
       </DemoDeploymentNote>
```

Comment:
We may want to consider internationalizing this `DemoDeploymentNote` at some point, in case we anticipate Spanish-dominant users using our demo site.